src/libutil/filesystem.cc

```diff
@@ -76,6 +77,7 @@ void createSymlink(const Path & target, const Path & link,
         if (lutimes(link.c_str(), times))
             throw SysError("setting time of symlink '%s'", link);
     }
+#endif
```

Comment:
That guard (and the one below which acts on essentially duplicated code :/ ) is a bit annoying because it changes the semantics and affects reproducibility. Do you know whether there's a way to emulate `lutimes` if it's absent?

Or maybe we can just ignore setting the time for the symlinks if `lutimes` isn't available and pray that it doesn't matter in practice

Reply:
> futimes() is available since glibc 2.3. lutimes() is available since glibc 2.6, and is implemented using the [utimensat](https://linux.die.net/man/2/utimensat)(2) system call, which is supported since kernel 2.6.22.

Reply:
@jugendhacker Does termux has the mentioned function ?